### PR TITLE
improves sidebar behaviour

### PR DESCRIPTION
**Pull request by lindseybateman**
screen shot 2015-06-17 at 15 30 47

### CODE REVIEW

**Comment by mies** [comment]
this doesn't really work for me or I'm doing something wrong (I built the app and regenerated the content, just in case). When going to the doc page I just have a search bar and no table of contents. See screenshot

screen shot 2015-06-17 at 15 47 07

**Comment by lindseybateman** [comment]
The behaviour of the sidebar has changed a bit. 
screen shot 2015-06-18 at 11 10 29
screen shot 2015-06-18 at 11 10 21

**Comment by flenter** [comment]
You can now collapse/expand items, right?

**Comment by Tonnu** [comment]
Ohh collapsable headers! Very nice :100:

**Comment by lindseybateman** [comment]
yes but only one at a time :)

**Comment by flenter** [comment]
Why only one? Is that a stylesheet issue?

File: docs-sidebar/search.css [hunk 1]
```diff
@@ -16,18 +19,44 @@
...
+.clear-search.show {
+  display: block;
+}
+
+.clear-search:hover {
+  cursor: pointer;
+  cursor: hand;
+}
+
+.clear-search:before {
```
Comment (yoshuawuyts) [inline docs-sidebar/search.css]: adding a comment as to what this is might be helpful.

**Comment by yoshuawuyts** [comment]
:ok_hand: dope

**Pull request merged**